lib/core/src/Cardano/Wallet/DB/Sqlite/TH.hs

```diff
@@ -227,11 +227,12 @@ UTxO                                sql=utxo
 -- Sequential scheme address discovery state
 -- which does not belong to a particular checkpoint.
 SeqState
-    seqStateWalletId        W.WalletId        sql=wallet_id
-    seqStateExternalGap     W.AddressPoolGap  sql=external_gap
-    seqStateInternalGap     W.AddressPoolGap  sql=internal_gap
-    seqStateAccountXPub     B8.ByteString     sql=account_xpub
-    seqStateRewardXPub      B8.ByteString     sql=reward_xpub
+    seqStateWalletId          W.WalletId         sql=wallet_id
+    seqStateExternalGap       W.AddressPoolGap   sql=external_gap
+    seqStateInternalGap       W.AddressPoolGap   sql=internal_gap
+    seqStateAccountXPub       B8.ByteString      sql=account_xpub
+    seqStateRewardXPub        B8.ByteString      sql=reward_xpub
+    seqStateDerivationPrefix  W.DerivationPrefix sql=derivation_prefix
```

Comment:
Maybe I'm being pedantic, but do we need to align all these rows?

Reply:
We don't need to. It helps readability, but is arguably superfluous.